dev/devfile.yaml

```diff
@@ -10,28 +10,25 @@ projects:
       location: 'https://github.com/OpenLiberty/application-stack.git'
       sparseCheckoutDir: /templates/default
 components:
-  - id: redhat/java/latest
-    memoryLimit: 1512Mi
-    type: chePlugin
-  - alias: devruntime
-    mountSources: true
-    type: dockerimage
-    image: kabanero/ubi8-maven:0.3.1 
-    memoryLimit: 1512Mi
-    volumes:
-    - name: m2
-      containerPath: /home/user/.m2
-    endpoints:
-    - name: 9080/tcp
-      port: 9080
-    - name: 9443/tcp
-      port: 9443
+- type: chePlugin
+  id: redhat/java/latest
+  memoryLimit: 1512Mi
+- alias: devruntime
+  mountSources: true
+  type: dockerimage
+  image: ajymau/java-openliberty-odo 
+  memoryLimit: 1512Mi
+  endpoints:
+  - name: 9080/tcp
+    port: 9080
+  - name: 9443/tcp
+    port: 9443
 commands:
   - name: devInit
     actions:
       - workdir: /projects/user-app
         type: exec
-        command: mvn -B liberty:install-server liberty:create liberty:install-feature dependency:go-offline
+        command: echo "moving liberty" && mkdir -p /projects/user-app/target/liberty && mv /opt/ol/wlp /projects/user-app/target/liberty 
```

Comment:
Since moving a populated 'target' is an unusual thing to do, I wonder if it'd be better to exclude the usr/servers/defaultServer and be a bit cleaner.